tests/multipleBackend/routes/routeBackbeat.js

```diff
@@ -292,16 +291,14 @@ describe('backbeat routes', () => {
             assert.strictEqual(StorageClass, 'STANDARD');
         }
 
-        beforeEach(done =>
+        beforeEach(() =>
             bucketUtil.emptyIfExists(BUCKET_FOR_NULL_VERSION)
                 .then(() => s3.createBucket({ Bucket: BUCKET_FOR_NULL_VERSION }).promise())
-                .then(() => done(), err => done(err))
         );
```

Comment:
i don't understand how this code "waits" for the bucket to be created :(
(it would be the case if promise was returned, but i don't see how it is here...)